externalpolicy/builtin_properties.go

```diff
@@ -27,7 +28,7 @@ const MAX_MEMEORY = 1048576 // the unit is MB. This is 1000G
 
 // get the node's built-in ptoperties to be used in the node policy
 // availableMem -- the total memory vs. the available memory size
-func CreateNodeBuiltInPolicy(availableMem bool) *ExternalPolicy {
+func CreateNodeBuiltInPolicy(availableMem bool, includeGenHwId bool) *ExternalPolicy {
```

Comment:
If the user does 'hzn policy update -f <new.policy.json>' where they omit the openhorizon.hardwareId in new.policy.json, and the hardwareId has to be generated (because there is no machine serial) then a new value is generated for hardwareId. That should not happen, the old value should remain the same after it is set for the first time.